CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 ##
 ##===----------------------------------------------------------------------===//
 
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.13.4)
```

Comment:
Do you need to bump this? How old is 3.13.4? Is it a version that most circt users would have?

Reply:
I took this from npcomp, but I'm not sure exactly why this version was used. 3.13 was released in Nov 2018, but I think 3.10 was what is available by default on ubuntu 18.04. I'll see, if it's not required I won't bump it.

Reply:
Actually I guess this version comes from upstream MLIR: https://github.com/llvm/llvm-project/blob/9e9599ef782384847e734dfdb89d34a4bb8c6710/mlir/CMakeLists.txt#L6. It's also the documented version for building LLVM, so I think it's generally safe to assume CIRCT developers will have this: https://llvm.org/docs/GettingStarted.html#software.